README.md

```diff
@@ -1,21 +1,75 @@
+
+### Data binding
+
+Accordion component provides an option to get the accordion items from the local data. It can be done through iteration of the Accordion Items using conditional foreach loop. Accordion only supports the local data for [data binding](https://blazor.syncfusion.com/documentation/accordion/data-binding?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples).
+
+### Icon support
+
+Add prefix icons or images to the panel header. The default expand and collapse icon used in the header is customizable.
+
+## Related links
+[Learn More about Blazor Accordion](https://www.syncfusion.com/blazor-components/blazor-accordion?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+[Documentation](https://blazor.syncfusion.com/documentation/accordion/getting-started?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+[View Demos](https://blazor.syncfusion.com/demos/accordion/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+[Online example](https://blazor.syncfusion.com/demos/accordion/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+[Video Tutorials](https://www.syncfusion.com/tutorial-videos/blazor/accordion?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+[Visual Studio 2019 (version 16.4+)](https://visualstudio.microsoft.com/downloads?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+[.NET Core SDK 3.1.201+](https://dotnet.microsoft.com/download/dotnet-core/3.1?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+[Knowledge base](https://www.syncfusion.com/kb/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples)
+
+## About Syncfusion Blazor Components
+The Syncfusion's Blazor components library offers over 70 UI components to work with Blazor server-side and client-side (Blazor WebAssembly) projects seamlessly. In addition to Accordion, we provide popular Blazor Components such as DataGrid, Charts, Scheduler, Diagram, Word Processor, and more.
```

Comment:
Need to add FT link for all controls
![image](https://user-images.githubusercontent.com/61148953/161201264-f4129288-2d57-4736-8a53-8aa50ac47410.png)